config/settings.py

```diff
@@ -91,16 +90,28 @@
 
 AUTH_PASSWORD_VALIDATORS = [
     {
-        "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",
+        "NAME": (
+            "django.contrib.auth.password_validation"
```

Comment:
Could you put these back in a single string, this is harder to read, and the new flake8 rules shouldn't complain.

Reply:
Also applies to below